recipes/mixins/_order.scss

```
@@ -0,0 +1,39 @@
+@function order_documentTitle($documentTitleConfig, $element) {
+    $contentValue: null;
+
+    @each $titlePart in map-keys($documentTitleConfig) {
+        $titleValue: map-get($documentTitleConfig, $titlePart);
+        $value: null;
+
+        @if $titleValue == 'partTypeTitleConfig' {
+            $value: pending(b#{$element}Label);
+        }
+        @if $titleValue == 'existingContent' {
```

```suggestion
        @else if $titleValue == 'existingContent' {
```

`$value` is not guaranteed to be not null after this block, but null is also not an acceptable value for `$value` given how it is used.

---

Perhaps also an `@else` block that errors when an acceptable `$titleValue` is not given would be nice as well.